articles/portal/ptl-ref-portal-api.md

```diff
@@ -212,593 +212,45 @@ Returns a response with a list of up to 10 VMs
 ```
 
 ## GET /api/my_calls
-
+#### With the move to the ITSM tool this functionality is no longer available.
```

Comment:
I'd recommend replacing the repeated use of this with a warning like the ones you see on pages like [this](https://docs.ukcloud.com/articles/other/other-ref-knowledge-guidelines.html) ([source markdown](https://github.com/UKCloud/documentation/blob/master/articles/other/other-ref-knowledge-guidelines.md)) It's possible to spin up a local instance of these docs using MonoDevelop and DocFX – consider checking the DocFX documentation to find out how to do this.